pkg/handlers/signup/provisioner/handlers.go

```diff
@@ -0,0 +1,164 @@
+package provisioner
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/labstack/echo/v4"
+	core "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/konflux-ci/workspace-manager/pkg/api/v1alpha1"
+)
+
+const (
+	UserEmailAnnotation string = "konflux-ci.dev/requester-email"
+	UserIdAnnotation    string = "konflux-ci.dev/requester-user-id"
+)
+
+func normalizeEmail(email string) string {
```

Comment:
I think the function name is misleading. It returns the namespace name rather than just normalizing the email.